src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,18 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.trace("Username or password are incorrect, ", e);
```

Comment:
The log message could be more informative by including parameters or user-relevant information. Consider adding the username to the log output to make it clearer which login attempt failed.